rs/boundary_node/rate_limits/canister/canister.rs

```diff
@@ -0,0 +1,23 @@
+use rate_limits_api::{GetConfigResponse, Version};
+use storage::VERSION;
+use types::{ConfigResponse, OutputConfig};
+mod storage;
+mod types;
+
+#[ic_cdk_macros::update]
+fn get_config(version: Option<Version>) -> GetConfigResponse {
```

Comment:
Friendly suggestion - you may not agree with it. Notice that canisters don't really have a `main` function, so it's hard to gauge "what runs first". This can make it tricky to have a clear understanding of the order of operations happening in the canister upon startup (the various initialization steps, like declaring stable memory, canister methods, etc). For that reason, my suggestion would be to not rush to break up the canister code into separate files. Instead, treat `canister/lib.rs` similarly to how you would treat your `main.rs` file, meaning configure all your initialization logic there sequentially, like you would do in a `main` function. So in this case:
```
# lib.rs

... define stable memory

... define dependencies that will be used in your canister methods

... define canister methods
```
then on a per-need basis you'd create new files to host things that get imported into your lib.rs. All this is mostly so that it's easier to understand the flow of the canister. As it is right now, you would have to ask yourself, what's happening first? Stable memory in storage.rs? Canister methods in canister.rs?

Reply:
> Notice that canisters don't really have a main function, so it's hard to gauge "what runs first". 

Yes, canister is an actor. 

>  For that reason, my suggestion would be to not rush to break up the canister code into separate files.

I understand. Since I have already prototyped the canister locally, i wanted  to have at least a minimum separation of files. For example `canister/types.rs` and `canister/canister.rs` (lib.rs) make sense already now IMO. Stable memory variables and `init` i can add in the next MR. Wdyt? However, I still wanted a working canister at this point.